debug-scripts/etcd

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -ux
+
+systemctl status etcd > $DEBUG_SCRIPT_DIR/etcd-systemctl-status
+journalctl -u etcd > $DEBUG_SCRIPT_DIR/etcd-journal
+
+alias etcdctl="etcdctl --cert-file /etc/ssl/etcd/client.crt --key-file /etc/ssl/etcd/client.key --ca-file /etc/ssl/etcd/ca.crt"
+etcdctl cluster-health > $DEBUG_SCRIPT_DIR/etcdctl-cluster-health
+etcdctl ls --recursive > $DEBUG_SCRIPT_DIR/etcdctl-ls
+etcdctl member list > $DEBUG_SCRIPT_DIR/etcdctl-member-list
```

Comment:
@chuckbutler I made a naive attempt to grab some useful info from etcd. Do you have any ideas what else we might want here?